### ISSUE TITLE
Unable to retreive nicehash profitability data

### ISSUE THREAD

**Issue by francescocmazza** [nicehash/NiceHashMiner#954]
Hello Dillon,
Often NHML has problem getting profitability data, even if internet connection is fine, as well as the connection to NH pools are fine.
Can you please find a way to set NHML to use latest valid profitability data if it is unable to retrieve latest? this way our miners don't stay down even if they get this bug.
Thank you

**Comment by yada-yoda** [comment]
Yes, having the same issue. On past versions if this was prompted it would still let you start mining, now on this version it puts a brick wall up. Not good!

Edit: Also as a side note, is there a static address if there is a hardware firewall that should be allowed to communicate? I don't think this is the issue but just addressing the question.

**Comment by hanyiabc** [comment]
Having this issue randomly. It makes my rigs only run 10% of the time. I have to check every hour. It used to have this problem only after several hours, now i can't even mine continuously for one hours.

**Comment by rigide** [comment]
PLEASE have an option to "Auto close and reopen NiceHash if not mining"

**Comment by pducharme** [comment]
+1 on the suggestion to have an "Auto-Close and reopen Nicehash".   Also, please add a simple way to copy over the Benchmarks of same model of cards.  (Select 1 and click the one you want to copy to, with a "copy benchmarks" button ?.

**Comment by borimol** [comment]
I have the same problem :(

**Comment by june-apiwat** [comment]
This is problem of internet service provider block ip to get profit info. I work around by connect with mobile internet 4G or whatever then start nicehash client, after finish loading switch back to fiber internet and start mining

**Comment by maurezen** [comment]
Seeing this issue a lt as well.

**Comment by utc99** [comment]
Yea, I thought this was solved on some versions.. I still get a lot of unable to get profitability data, and I'm from EU, don't see why ip's or something should be blocked. Idea to use latest good profitability data if there was a problem updating it might solve the problem a bit, and does not seem that it would be that hard to implement it, what do you think  @DillonN ?
![image](https://user-images.githubusercontent.com/26816096/39857297-80c40cc6-543b-11e8-815b-3177bcec3f05.png)

**Comment by S74nk0** [comment]
Issue related to old nhm version/build. Please use latest version and open a new issue for any bugs or feature requests.

Temporary back-end service issue not related to miner client
**OR**
Networking issue on your ISP side or ISP is blocking our services
